go.mod

```diff
@@ -8,6 +8,7 @@ go 1.22.0
 require (
 	github.com/AzureAD/microsoft-authentication-library-for-go v1.3.3
 	github.com/go-test/deep v1.1.1
+	github.com/golang-jwt/jwt/v5 v5.0.0
```

Comment:
Hmm, this is relatively old. Not necessarily a problem, but noticed it.

Reply:
agreed, upgraded to v5.2.1